src/Microsoft.Extensions.SecretManager.Tools/Internal/CommandLineOptions.cs

```diff
@@ -33,6 +34,11 @@ public static CommandLineOptions Parse(string[] args, TextWriter output)
             var optionProject = app.Option("-p|--project <PROJECT>", "Path to project, default is current directory",
                 CommandOptionType.SingleValue, inherited: true);
 
+            // the escape hatch if project evaluation fails, or if users want to alter a secret store other than the one
+            // in the current project
+            var optionId = app.Option("--id", "The user secret id to use.",
```

Comment:
> When would project evaluation fail?

At the moment, the question is "When does project evaluation **not** fail?" :trollface: 

Bad XML, missing imports, invalid syntax, etc... there are lots of reasons why it's good to have a an escape hatch. Plus, some may prefer to just us `--id` and avoid the MSBuild complexity.